lib/systems/default.nix

```diff
@@ -98,7 +98,7 @@ rec {
       in
         if final.parsed.kernel.name == pkgs.stdenv.hostPlatform.parsed.kernel.name &&
            (final.parsed.cpu.name == pkgs.stdenv.hostPlatform.parsed.cpu.name ||
-            (final.platform.isi686 && pkgs.stdenv.hostPlatform.isx86_64))
+            ((final.platform.isi686 or false) && (pkgs.stdenv.hostPlatform.isx86_64 or false)))
```

Comment:
cc @matthewbauer

Reply:
Yeah that should just be final.isi686

Reply:
Actually, because we want to eventually support WSL, we may want to not do this at all. WSL runs on x86-64 but will not run i686 binaries (https://github.com/Microsoft/WSL/issues/2468).